contracts/drafts/issuance/Issuance.sol

```diff
@@ -164,6 +168,15 @@ contract Issuance is Ownable, StateMachine, ReentrancyGuard {
         _transition("FAILED");
     }
 
+    /**
+     * @dev Function to transfer all collected tokens to the wallet of the owner
+     */
+    function transferFunds() public onlyOwner {
+        require(currentState == "LIVE", "Cannot transfer funds now.");
+        currencyToken.transfer(wallet, amountRaised);
+        emit FundsTransferred();
```

Comment:
You don't need this event. The only state change due to `transferFunds` is inside the `transfer` call which emits its own method.